geomstats/geometry/grassmannian.py

```diff
@@ -78,3 +84,34 @@ def exp(self, vector, point):
         expm = gs.linalg.expm
         mul = MatricesSpace.mul
         return mul(mul(expm(vector), point), expm(-vector))
+
+    def log(self, point, base_point):
+        """
```

Comment:
Do you have references / formulas for the expressions of the log and the exp that we could put in the docstring?

Reply:
Will do for the exp soon!